README.md

```diff
@@ -142,14 +143,24 @@ cd vcpkg
 The libheif port in vcpkg is kept up to date by Microsoft team members and community contributors. If the version is out of date, please [create an issue or pull request](https://github.com/Microsoft/vcpkg) on the vcpkg repository.
 
 
+### Adding libaom encoder/decoder for AVIF
+
+* Run the `aom.cmd` script in the `third-party` directory to download libaom and
+  compile it.
+
+When running `cmake` or `configure`, make sure that the environment variable
+`PKG_CONFIG_PATH` includes the absolute path to `third-party/aom/dist/lib/pkgconfig`.
+
+
 ### Adding rav1e encoder for AVIF
 
 * Install `cargo`.
 * Install `cargo-c` by executing
 ```
 cargo install --force cargo-c
 ```
-* Run the `rav1e.cmd` script in directory `third-party` to download rav1e and compile it.
+* Run the `rav1e.cmd` script in the `third-party` directory to download rav1e
+  and compile it.
```

Comment:
In this change and the changes below, I changed 
```
directory `third-party`
```
to
```
the `third-party` directory
```

The latter sounds better.